### PR TITLE
Use ws_ping_interval to avoid disconnects

### PR DESCRIPTION
**Pull request by pablogsal**

### CODE REVIEW

File: master/master.cfg [hunk 1]
```diff
@@ -379,6 +379,7 @@ c["www"] = dict(
         },
     },
     plugins=dict(waterfall_view={}, console_view={}, grid_view={}),
+    ws_ping_interval=30,
```
Comment (vstinner) [inline master/master.cfg:382]: Maybe add a comment to https://bugs.python.org/issue41701

Comment (pablogsal) [reply master/master.cfg:382]: I still need to figure out how to update the server

**Comment by pablogsal** [comment]
I'm going to merge this to try out if this fixes the problem, we can revert it later if it doesn't

**Comment by pablogsal** [comment]
Unfortunately, there are several problems with buildbot 3.10.1+ so I had to downgrade: https://github.com/python/buildmaster-config/pull/240 :(

**Pull request merged**